app/src/main/java/de/westnordost/streetcomplete/quests/toilet_availability/AddToiletAvailability.java

```diff
@@ -36,10 +63,89 @@ public void applyAnswerTo(Bundle answer, StringMapChangesBuilder changes)
 		changes.add("toilets", yesno);
 	}
 
+	@Nullable @Override public Boolean isApplicableTo(Element element) { return null; }
+
+	@Override public boolean download(BoundingBox bbox, final MapDataWithGeometryHandler handler)
+	{
+		List<AddHousenumber.ElementWithGeometry> items = downloadElementsWithoutToilets(bbox);
```

Comment:
Wrong name in the long run :)

Reply:
Do you have another suggestion?

Reply:
> Do you have another suggestion?

Make the class not a inner class of AddHousenumber.